index.html

```diff
@@ -1,189 +1,319 @@
-                        <ul class="flex">
-                            <li>Codekit</li>
-                            <li>Github</li>
-                            <li>Codepen</li>
-                            <li>Gitlab</li>
-                            <li>Terminal</li>
-                        </ul>
-                    </li>
-                </ul>
-        </section>
-</body>
-</html>
\ No newline at end of file
+      <ul class="lang_skill_frame flex">
+        <li class="cate_list flex">
+          <div class="category_heading flex">
+            <img
+              class="rect_rotate"
+              src="https://res.cloudinary.com/david79/image/upload/v1674029967/work%20stuff/Rectangle-56_czhvo3.png"
+              alt="list style"
+            />
+            <h3>Language</h3>
+          </div>
+          <ul class="flex">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </li>
+        <li class="cate_list flex">
+          <div class="category_heading flex">
+            <img
+              src="https://res.cloudinary.com/david79/image/upload/v1674029967/work%20stuff/Rectangle-56_czhvo3.png"
+              alt="list style"
+            />
+            <h3>Frameworks</h3>
+          </div>
+          <ul class="flex">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </li>
+        <li class="cate_list flex">
+          <div class="category_heading flex">
+            <img
+              src="https://res.cloudinary.com/david79/image/upload/v1674029969/work%20stuff/Rectangle-57_q7fssx.png"
+              alt="list style"
+            />
+            <h3>Skills</h3>
+          </div>
+          <ul class="flex">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </li>
+      </ul>
+    </section>
+    <section id="contact" class="flex">
+      <h4>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h4>
+      <form action="https://formspree.io/f/mrgvbrjo" method="post" class="flex">
+        <input
+          type="text"
+          name="full_name"
+          id="full_name"
+          placeholder="Full name"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="Email address"
+          required
+        />
+        <textarea
+          name="message"
+          id="message"
+          cols="30"
+          rows="10"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        ></textarea>
+        <input type="submit" value="Get in touch" class="contact_btn btn" />
```

Comment:
- [ ] Great job with everything 👏 You only missed one part about the design which is the button interaction styles as shown in Figma 👍 

![image](https://user-images.githubusercontent.com/50721479/214656445-94a8b78b-5ff4-461a-8b25-4e3d3bf22ee2.png)